XLForm/XL/Helpers/UIView+XLFormAdditions.m

```diff
@@ -27,47 +27,60 @@
 
 @implementation UIView (XLFormAdditions)
 
-+ (id)autolayoutView
++ (instancetype)autolayoutView
 {
-    UIView *view = [self new];
+    __kindof UIView *view = [self new];
     view.translatesAutoresizingMaskIntoConstraints = NO;
+    
     return view;
 }
 
--(NSLayoutConstraint *)layoutConstraintSameHeightOf:(UIView *)view
+- (NSLayoutConstraint *)layoutConstraintSameHeightOf:(UIView *)view
 {
-    return [NSLayoutConstraint constraintWithItem:self attribute:NSLayoutAttributeHeight relatedBy:NSLayoutRelationEqual toItem:view attribute:NSLayoutAttributeHeight multiplier:1.0f constant:0.0f];
+    return [NSLayoutConstraint constraintWithItem:self
+                                        attribute:NSLayoutAttributeHeight
+                                        relatedBy:NSLayoutRelationEqual
+                                           toItem:view
+                                        attribute:NSLayoutAttributeHeight
+                                       multiplier:1.0
+                                         constant:0.0];
 }
 
 - (UIView *)findFirstResponder
 {
+    UIView *firstResponder = nil;
     if (self.isFirstResponder) {
-        return self;
+        firstResponder = self;
     }
     for (UIView *subView in self.subviews) {
-        UIView *firstResponder = [subView findFirstResponder];
-        if (firstResponder != nil) {
-            return firstResponder;
+        UIView *fr = [subView findFirstResponder];
```

Comment:
Why di you change the name of the variable. `firstResponder` is better to understand here